### PR TITLE
Added supporting non ascii symbols #12

### PR DESCRIPTION
**Pull request by WindowGenerator**
Issue #12 

- Formatted all code with `cargo fmt`
- Fixed some tests related to non-ascii chars
- Added non-ascii supporing

image

### CODE REVIEW

File: src/expected_input.rs [hunk 1]
```diff
@@ -206,4 +207,12 @@ mod tests {
             "At least 4 items contain only digits"
         );
     }
+    #[test]
+    fn shoudl_work_with_non_ascii_chars() {
```
Comment (radlinskii) [inline src/expected_input.rs:211]: typo in "should"

**Review by radlinskii — COMMENTED** [review]
Great job! looks clean! 👏 

I'm putting `hacktoberfest-accepted` label, but if you want to merge it please resolve the conflicts first.

File: src/expected_input.rs (hunk 1 above)
Comment (WindowGenerator) [reply src/expected_input.rs:211]: ohh sorry, didn't notice

**Review by radlinskii — APPROVED** [review]
LGTM, thanks!

**Pull request merged**